### ISSUE TITLE
Update internal POC search to include all MDWG members

### ISSUE THREAD

**Issue by grioux-neo** [cap-md089/evmplus-v6#123]
**Is your feature request related to a problem? Please describe.**
In operation many, if not most, wing events have a POC who is not a member of the wing staff.  In the current system, in order to provide appropriate permissions for those members to manipulate the event items, managers must have direct access to all members' CAPID.  Most managers do not have access to those data.

**Describe the solution you'd like**
The 'add POC' interface should be expanded to allow searchable selection across all members in the database.

**Comment by r-a303931** [comment]
Good idea, but probably not fit for v6. There exists the 101 card lookup on eServices which allows looking up by name different people and shows CAP IDs, as well as unit number and wing. It's clunky, but it works and I'm willing to bet people at the Wing level and above are familiar with it.

We also have to take into account performance; it currently takes ~6 seconds to load data for St. Mary's at 100 members; it we expanded that to the wing at around 1200 members, that would increase loading times to over a minute. Even if we filter out cadets, that would still bring load times to around 40 seconds. The alternative is to make a paging system, which is complex and wouldn't fit within the schedule we have for v7.

**Comment by grioux-neo** [comment]
Let's discuss options.  I don't think that waiting for an entire rewrite
(it took 18 months to release the last rewrite) is reasonable to hold off
on this, especially once use is mandated by the wing.
